shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -1262,6 +1263,7 @@ private void sendViewportMetricsToFlutter() {
     }
 
     viewportMetrics.devicePixelRatio = getResources().getDisplayMetrics().density;
+    viewportMetrics.physicalTouchSlop = ViewConfiguration.get(getContext()).getScaledTouchSlop();
```

Comment:
https://github.com/flutter/engine/blob/275066a84e0b666df9acd3758dd8f931ef38676e/shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java#L240-L241 shows how to get the viewport metrics from FlutterView in case you want to unit test it.